R-package/src/install.libs.R

```diff
@@ -15,6 +15,39 @@ if (!(R_int_UUID == "0310d4b8-ccb1-4bb8-ba94-d36a55f60262"
   print("Warning: unmatched R_INTERNALS_UUID, may cannot run normally.")
 }
 
+.find_vs_version <- function() {
+  current_working_dir <- getwd()
+  on.exit({
+    setwd(current_working_dir)
+  })
```

Comment:
Should we also remove `build_dir`?

Reply:
Each R session creates a temporary directory that is attached to the session and destroyed whenever the session exits. `tempdir()` creates a directory under that one, so it also automatically gets destroyed.

Reply:
not bad for a programming language invented by and for statisticians, huh 😀